sagemaker-training-compiler/huggingface/pytorch_multiple_gpu_multiple_node/language-modeling-multi-gpu-multi-node.ipynb

```diff
@@ -2,30 +2,30 @@
  "cells": [
```

Comment:
<p>Provide justification for why we need these libraries in this environment. I don't see <code>torch</code> being used anywhere. Same for <code>s3fs.</code> Please prune this list.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3611/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>